aten/src/ATen/core/jit_type.h

```diff
@@ -308,15 +308,15 @@ struct CAFFE2_API TensorType : public DynamicType {
 
   at::ScalarType scalarType() const { return scalar_type_; }
   at::Device device() const { return device_; }
-  int dim() const { return dim_; }
+  size_t dim() const { return dim_; }
```

Comment:
Are you sure about this? All of ATen uses `int64_t` for sizes and anything integral, so this might cause problems